pyccel/parser/semantic.py

```diff
@@ -2505,6 +2497,8 @@ def _visit_FunctionDef(self, expr, **settings):
             arguments      = expr.arguments
             header_results = m.results
 
+            if len(interfaces) > 1:
+                name = interface_name + '_' + str(i)
```

Comment:
Have you thought about formatting this so you always get a name with the same length? e.g.:
```
interface_name_01
interface_name_02
...
interface_name_12
```
instead of:
```
interface_name_1
interface_name_2
...
interface_name_12
```